dashboard/app/views/shared/_course_progress_block.html.haml

```diff
@@ -1,19 +1,16 @@
 - user_script ||= nil
-.course_progress
+.course_progress{style: 'float: left; position: relative;'}
   - unless script.name.in?([Script::HOC_NAME, Script::FROZEN_NAME, Script::INFINITY_NAME, Script::FLAPPY_NAME, Script::PLAYLAB_NAME, Script::ARTIST_NAME, Script::COURSE1_NAME, Script::COURSE2_NAME, Script::COURSE3_NAME, Script::COURSE4_NAME, Script::TWENTY_HOUR_NAME])
-    .row#lesson
-      .span9#heading
-        %h3.whitetext{style: 'margin: 0; display: inline-block'}
-          = image_tag(script.logo_image, class: 'logo') if script.logo_image
-          = data_t_suffix('script.name', script.name, "title")
-        - if script.beta?
-          %span.betatext{style: 'font-size: 24.5px'}
-            != '&nbsp; beta'
-
-      .span3#heading{style: 'text-align: right;'}
+    .courseblock-span3.courseblock-wide-small{style: 'width: 460px; height: 110px;'}
+      =link_to script_url(script.id) do
+        .heading
+          %h3= data_t_suffix('script.name', script.name, 'title')
+        .clear
+        .smalltext= data_t_suffix('script.name', script.name, 'description')
+        .clear
         - if user_script.try(:completed_at)
```

Comment:
for those users who don't have completion recorded properly, you could maybe do this:

- if user_script.try(:completed_at) || (user_script && next_unpassed_progression_level.nil?)

see similar in https://github.com/code-dot-org/code-dot-org/pull/2944

optional for now.